STM32Cube/Tasks/state_estimation.c

```diff
@@ -15,35 +15,39 @@
 #include "printf.h"
 #include "state_estimation.h"
 
-#define TASK_DELAY_TICKS 20 // TODO: what is actual delay time?
-#define SAMPLE_RATE 100 // replace this with actual sample rate
-#define USE_ICM 0
+#define SAMPLE_RATE_HZ 1 //Hz; Need to set to match VN data rate
```

Comment:
1 Hz ??

Reply:
Yes. Until we do some more testing with bumping the VN output rate, this is the config that works (VN is outputting at nominal 2 Hz but I ran into issues with readings getting dropped so this is what works consistently)